src/business-logic/facades/service-facade.ts

```diff
@@ -1,26 +1,23 @@
 import { RundownService } from '../services/interfaces/rundown-service'
-import { RundownEventService } from '../services/rundown-event-service'
 import { RundownTimelineService } from '../services/rundown-timeline-service'
 import { RepositoryFacade } from '../../data-access/facades/repository-facade'
 import { TimelineBuilder } from '../services/interfaces/timeline-builder'
 import { SuperflyTimelineBuilder } from '../services/superfly-timeline-builder'
-import { RundownEventBuilderImplementation } from '../services/rundown-event-builder-implementation'
-import { RundownEventBuilder } from '../services/interfaces/rundown-event-builder'
 import { TimeoutCallbackScheduler } from '../services/timeout-callback-scheduler'
 import { JsonObjectCloner } from '../services/json-object-cloner'
 import { BlueprintsFacade } from '../../blueprints/blueprints-facade'
 import { ActionService } from '../services/interfaces/action-service'
 import { ExecuteActionService } from '../services/execute-action-service'
+import { EventEmitterFacade } from '../../presentation/facades/event-emitter-facade'
```

Comment:
Just leaving a comment so people are aware that we are going to be very strict about how much of the presentation layer the service layer is allowed to know.

Only the facades in the service layer are allowed to know anything about the presentation layer and the only think they are allowed to know about is facades in the presentation layer.
Further more, we are going to limit how many places a Service layer facade can call a presentation layer facade.

We currently _only_ allow this one place. And there needs to be a _strong_ argument for letting another place know about it too.